sourceCode.py

```diff
@@ -1 +1,515 @@
+            if match:
+                resp = random.choice(values_cursing[i])
+                pos = resp.find('%')
+                print(resp)
+                else:
+                    print("I did not understand what you said")
+                    
+######################################
+##                                  ##
+## Talking with Jordan about Corona ##
+##                                  ##
+######################################
+
+## create library for talking about Corona
+library_corona = [
+    [r'Hi',
+     [ "Hey. How do you feel about Corona?",
+      "Hey, do you want to talk about Corona?" ]],
+    
+    [r'What can I do to stay sane?',
+     [  "You could go out for a walk.",
+      "You could meet someone with 1.5 meters distance.",
+      "You could talk to a friend",
+      "Doing some sports could help as well!"]],
+        
+    [r'(.*) suggestions (.*)',
+     [  "You could go out for a walk.",
+      "You could meet someone with 1.5 meters distance.",
+      "You could talk to a friend"
+      "Doing some sports could help as well!"]],
+    
+    [r'(.*) insane (.*)',
+     [  "I understand. This is a special situation.",
+      "How are you currently handling it?",
+      "Do you have someone to talk to except me?."]],
+
+    [r'I don\'?t (.*)',
+     [  "Don't you really %1?",
+      "Why don't you %1?",
+      "Do you want to %1?"]],
+    
+    [r'I feel (.*)',
+     [  "Good, tell me more about these feelings.",
+      "Do you often feel %1?",
+      "When do you usually feel %1?",
+      "When you feel %1, what do you do?"]],
+    
+    [r'I have (.*)',
+     [  "Why do you tell me that you've %1?",
+      "Have you really %1?",
+      "Now that you have %1, what will you do next?"]],
+    
+    [r'(.*)\?',
+     [  "Why do you ask that?",
+      "Perhaps the answer lies within yourself?",
+      "Why don't you tell me?"]],
+    
+    [r'I would (.*)',
+     [  "Could you explain why you would %1?",
+      "Why would you %1?",
+      "Who else knows that you would %1?"]],
+    
+    [r'Is there (.*)',
+     [  "Do you think there is %1?",
+      "It's likely that there is %1.",
+      "Would you like there to be %1?"]],
+    
+    [r'quit',
+     [  "Thank you for talking with me, and #staysafe!",
+      "Good-bye, #staysafe"]],
+    
+## create function for cursing with Jordan
+import random
+import re
+
+def corona_Jordan(userInput):
+    keys_corona = list(map(lambda x:re.compile(x[0], re.IGNORECASE), library_corona))
+    values_corona = list(map(lambda x:x[1],library_cursing))
+    
+    while True:
+        userInput = input(">>> ")
+        for i in range(0, len(keys_corona)):
+            match = keys_corona[i].match(userInput)
+            if match:
+                resp = random.choice(values_corona[i])
+                pos = resp.find('%')
+                print(resp)
+                else:
+                    print("I did not understand what you said")
```

Comment:
Hi Marla, I think you are doing something pretty complicated and ambitious. It's a really cool project. Besides my minor comments, I think you are doing great! Curious what the final product will look like! I am definitely going to try out to talk to Jordan. 👍